Makefile-tests.am

```diff
@@ -20,38 +20,39 @@ endif
 
 GITIGNOREFILES += ssh-config ansible-inventory.yml vmcheck-logs/ test-compose-logs/ tests/vmcheck/image.qcow2
 
-testbin_cppflags = $(AM_CPPFLAGS) -I $(srcdir)/src/lib -I $(srcdir)/src/libpriv -I $(srcdir)/libglnx -I $(srcdir)/tests/common
-testbin_cflags = $(AM_CFLAGS) -fvisibility=hidden $(PKGDEP_RPMOSTREE_CFLAGS)
-testbin_ldadd = $(PKGDEP_RPMOSTREE_LIBS) librpmostree-1.la librpmostreepriv.la -lstdc++
-
-noinst_LTLIBRARIES += libtest.la
-libtest_la_SOURCES = tests/common/libtest.c
-libtest_la_CPPFLAGS = $(testbin_cppflags)
-libtest_la_CFLAGS = $(testbin_cflags)
-libtest_la_LIBADD = $(testbin_ldadd)
-
-tests_check_jsonutil_CPPFLAGS = $(testbin_cppflags)
-tests_check_jsonutil_CFLAGS = $(testbin_cflags)
-tests_check_jsonutil_LDADD = $(testbin_ldadd) libtest.la
-
-tests_check_postprocess_CPPFLAGS = $(testbin_cppflags)
-tests_check_postprocess_CFLAGS = $(testbin_cflags)
-tests_check_postprocess_LDADD = $(testbin_ldadd) libtest.la
-
-tests_check_test_utils_CPPFLAGS = $(testbin_cppflags)
-tests_check_test_utils_CFLAGS = $(testbin_cflags)
-tests_check_test_utils_LDADD = $(testbin_ldadd) libtest.la
-
-tests_check_test_sysusers_CPPFLAGS = $(testbin_cppflags)
-tests_check_test_sysusers_CFLAGS = $(testbin_cflags)
-tests_check_test_sysusers_LDADD = $(testbin_ldadd) libtest.la
-
-uninstalled_test_programs = \
-	tests/check/jsonutil			\
-	tests/check/postprocess			\
-	tests/check/test-utils			\
-	tests/check/test-sysusers		\
-	$(NULL)
```

Comment:
Hmm, is there *any* way we can keep these working somehow until we do move them to Rust? Doesn't feel great to just disable all these.

Reply:
I guess one potential path is https://github.com/coreos/rpm-ostree/pull/2393. Could another path be to just have CI CI rebuild a separate fat binary for each? (I guess we could merge all the tests in one so it just has to generate a single one.)

Reply:
So dbus [has a compile time option](https://gitlab.freedesktop.org/dbus/dbus/-/blob/669e3c0acabd58ac0c86146fe77022a54558966c/configure.ac#L235) - we could do that.

But most of this stuff we're not changing anytime soon, and I think we just want to sprint towards Rust.  For example I think we're not far already from having the stuff in `test-utils.c` be all implemented in Rust, then that whole file could be deleted.